deployctl.ts

```diff
@@ -7,7 +7,7 @@ import {
   semverParse,
   setColorEnabled,
 } from "./deps.ts";
-import { Args, parseArgs } from "./src/args.ts";
+import { type Args, parseArgs } from "./src/args.ts";
```

Comment:
Can you explain a bit this?

Reply:
A new lint rule was added to warn imports that are used only as types but imported without any modifier.
https://lint.deno.land/?q=verba&all=on

Reply:
I just ran deno lint with --fix option to get all of this fixed.